package.json

```diff
@@ -25,12 +25,13 @@
   "devDependencies": {
     "event-stream": "^3.1.7",
     "mocha": "~1.16.2",
+    "slash": "^1.0.0",
```

Comment:
Can be installed as non-dev dependency, because it will be used to resolve Windows style paths. Moving this down will make it easier to merge.